tests/llms/test_cohere.py

```diff
@@ -0,0 +1,149 @@
+from typing import Any
+
+import pytest
+from llama_index.llms.base import ChatMessage
+from pytest import MonkeyPatch
+
+try:
+    import cohere
+    from llama_index.llms.cohere import Cohere
```

Comment:
I believe this `Cohere` import doesn't need to be in the `try-Except`, just the `import cohere` line

Reply:
You are right. I'll update the PR with this